themes/Delicious/foundation/spacing.scss

```diff
@@ -1,13 +1,13 @@
 $unit: 4px;
 
 $spacing-data: (
-  extra-tight: rem($unit),
-  tight: rem($unit * 2),
+  tight: rem($unit),
+  tighter: rem($unit * 2),
+  tightest: rem($unit * 3),
```

Comment:
@Farhan-emgage this should be
```
tightest: rem($unit),
tighter: rem($unit * 2),
tight: rem($unit * 3),
```
Just as it is src/styles/foundation/spacing.scss